PointOfDispensingApp/src/brokerRequests.js

```diff
@@ -0,0 +1,35 @@
+import axios from "axios";
+
+// define functions for API requests here
+export default {
+  // Search patient for patient retrieval
+  searchPatient: (data) => {
+    let query = `?family=${data.lastName}&given=${data.firstName}&birthdate=${data.birthDate}&address-postalcode=${data.postalCode}`;
+    return axios.get(`http://localhost:3000/Patient${query}`);
+  },
+
+  // Get patient from QR code for patient retrival
+  getPatient: (data) => {
+    const patient = {
+      id: data.id,
+      family: "Fraley",
+      given: "David",
+      birthDate: "1950-01-01",
+      gender: "Male",
+      address: {
+        line: "1234 Main Street",
+        city: "Waukesha",
+        state: "WI",
+        postalCode: "53072",
+      },
+      language: "English",
+    };
+
+    return axios.get(`http://localhost:3000/healthcheck`).then((response) => {
```

Comment:
General for all axios calls. Should also handle errors with a catch() - https://www.intricatecloud.io/2020/03/how-to-handle-api-errors-in-your-web-app-using-axios/

Reply:
I've added some basic error handling that will need to be built upon later.